File: android/app/src/main/java/com/dtools/ReactPackage/ReactDeviceOwnerPackage.java
package com.dtools.ReactPackage;

import com.facebook.react.*;
import com.facebook.react.bridge.*;
import com.facebook.react.uimanager.*;
import java.util.*;

public class ReactDeviceOwnerPackage implements ReactPackage {

    @Override
    public List<Class<? extends JavaScriptModule>> createJSModules() {
        return Collections.emptyList();
    }

    @Override
    public List<ViewManager> createViewManagers(ReactApplicationContext reactContext) {
        return Collections.emptyList();
    }

    @Override
    public List<NativeModule> createNativeModules(ReactApplicationContext reactContext) {
        List<NativeModule> modules = new ArrayList<>();
        modules.add(new ReactDeviceOwnerManager(reactContext));
        return modules;
    }

}

File: android/app/src/main/kotlin/com/dtools/ReactPackage/ReactDeviceOwnerPackage.kt
package com.dtools.ReactPackage

import android.view.View
import com.facebook.react.*
import com.facebook.react.bridge.*
import com.facebook.react.uimanager.*
import java.util.*

class ReactDeviceOwnerPackage : ReactPackage {

    override fun createJSModules(): List<Class<out JavaScriptModule>> {
        return emptyList<Class<out JavaScriptModule>>()
    }

    override fun createViewManagers(reactContext: ReactApplicationContext): List<ViewManager<View, ReactShadowNode>> {
        return emptyList<ViewManager<View, ReactShadowNode>>()
    }

    override fun createNativeModules(reactContext: ReactApplicationContext): List<NativeModule> {
        val modules = ArrayList<NativeModule>()
        modules.add(ReactDeviceOwnerManager(reactContext))
        return modules
    }

}
